services/web/server/src/simcore_service_webserver/users/_users_rest.py

```diff
@@ -172,7 +172,7 @@ async def search_users_for_admin(request: web.Request) -> web.Response:
         UsersForAdminSearchQueryParams, request
     )
 
-    found = await _users_service.search_users(
+    found = await _users_service.search_users_as_admin(
```

Comment:
does that mean you search as an admin would do or you search users that are registered as admin?
I thought it was clear, but then I see the function above is `for_admin` so I am now a bit confused.

Reply:
yes, it is indeed not clear. 

The idea here is that by default all functions in in the service layer need to check whether the caller has enough access rights to execute the function. For that reason they have a signature like `def service_layer_funcion(product_name: ProductName, user_id: UserID, ... )`. 
Nonetheless, there are some situation in which we want to skip that step (e.g. when the background-task execute a function it has no user associated) and therefore those function do not include any information of the caller (e.g. user) or its context (e.g. product)